docs/overview/README.md

```diff
@@ -1,18 +1,27 @@
 # Overview
 
-The IOTA client library is a stateless library that provides a high level abstractions on top of IOTA node software to help interact with IOTA network in user friendly way and allows you to do the following, for instance:
+To communicate with the IOTA network, you have to connect and interact with a [node](https://chrysalis.docs.iota.org/node-software/node-software.html). This library, the IOTA Client, wraps routine tasks and minute details into its tidy API. It will generate addresses for you, send messages, sign and send transactions, and more.
 
-- Create messages and transactions
-- Sign transactions
-- Generate addresses
-- Interact with an IOTA node
+Beyond establishing the initial connection to a node, Client has no state. Operations use only the data that you pass during the call and have no effect on your code or the client instance beyond returning a value. You are in full control of the data flow in your application.
```

Comment:
```suggestion
Beyond establishing the initial connection to a node, the Client is stateless. Operations only use the data that you pass during the call and have no effect on your code or the client instance beyond returning a value. You are in full control of the data flow in your application.
```

Reply:
There is some controversy brewing among the devs whenever we call it stateless or not.

Isn't the initial version the correct one? "Operations only use the data" - operations only use data but don't do anything else with it, "Operations use only the data that you pass" - operations use the data that you pass and nothing else, which is what I intended to say. The former version could mean both, and the latter is not ambiguous.

Reply:
That checks out, the stateless thing I mean. It pops up in content from time to time.

And yes, with the clarification, that helps; keep the OG version!